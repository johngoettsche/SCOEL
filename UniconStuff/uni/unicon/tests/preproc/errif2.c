/* error message test: octal escape too large in a condition */

#if 0
one
#elif '\777'
two
#endif
