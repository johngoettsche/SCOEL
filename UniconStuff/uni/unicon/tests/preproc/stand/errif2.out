pp: File errif2.c; Line 5: octal escape sequece larger than a character
